Editor/Resources/EditorWindow/Pages/AnywherePage.uxml

```diff
@@ -0,0 +1,102 @@
+                <ui:Label name="AnywherePageConnectFleetIDDisplay" class="form-row__text-display" text="1234567890"/>
+            </ui:VisualElement>
+            <ui:VisualElement name="AnywherePageConnectFleetStatus" class="form-row form-row__hidden">
+                <ui:Label name="AnywherePageConnectFleetStatusLabel" class="form-row__label" text="Fleet status"/>
+                <ui:Label name="AnywherePageConnectFleetStatusDisplay" class="form-row__text-display" text="Connected"/>
+            </ui:VisualElement>
+            <ui:VisualElement name="AnywhereFleetButtons" class="form-row form-row__hidden">
+                <ui:VisualElement class="form-row__input form-row__input--no-label">
+                    <ui:Button name="AnywherePageConnectFleetNewButton" class="button button--primary" text="Create New Anywhere Fleet"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+        </ui:Foldout>
+        <ui:Foldout name="AnywherePageComputeTitle" class="foldout" text="Register Compute">
+            <ui:VisualElement class="form-row">
+                <ui:Label name="AnywherePageComputeNameLabel" class="form-row__label" text="Compute Name"/>
+                <ui:TextField name="AnywherePageComputeNameInput" class="form-row__input text-input" value="ComputerName-ProfileName"/>
+            </ui:VisualElement>
+            <ui:VisualElement class="form-row">
+                <ui:Label name="AnywherePageComputeIPLabel" class="form-row__label" text="IP Address"/>
+                <ui:VisualElement class="form-row__input form-row__input--inline">
+                    <ui:TextField name="AnywherePageComputeIpAddress" value="120" max-length="3" keyboard-type="NumberPad" class="text-input text-input--tiny text-input--centered"/>
+                    <ui:Label text="." class="ip-spacer"/>
+                    <ui:TextField name="AnywherePageComputeIpAddress" value="120" max-length="3" keyboard-type="NumberPad" class="text-input text-input--tiny text-input--centered"/>
+                    <ui:Label text="." class="ip-spacer"/>
+                    <ui:TextField name="AnywherePageComputeIpAddress" value="120" max-length="3" keyboard-type="NumberPad" class="text-input text-input--tiny text-input--centered"/>
+                    <ui:Label text="." class="ip-spacer"/>
+                    <ui:TextField name="AnywherePageComputeIpAddress" value="120" max-length="3" keyboard-type="NumberPad" class="text-input text-input--tiny text-input--centered"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+            <ui:VisualElement name="AnywherePageComputeStatus" class="form-row">
+                <ui:Label name="AnywherePageComputeStatusLabel" class="form-row__label" text="Status"/>
+                <ui:Label name="AnywherePageComputeStatusDisplay" class="form-row__text-display" text="Registered"/>
+            </ui:VisualElement>
+            <ui:VisualElement class="form-row">
```

Comment:
Should this visual element have a name too? That being said i see certain elements don't have the name property attached to them? How do we decide that? Is it only required for elements that are referenced elsewhere in code?

Reply:
Yes it is only required if we need to query the element in code